boards/b-l072z-lrwan1/Makefile.include

```diff
@@ -2,6 +2,8 @@
 export CPU = stm32l0
 export CPU_MODEL = stm32l072cz
 
+OPENOCD_STM32L0_DUAL_BANK ?= 1
```

Comment:
I would do it the other way around: disabled by default so users with stable openocd versions (0.10 but without dual bank support) won't have problems with this board.